.github/workflows/eslint_plugin_e2e_tests.yml

```
@@ -0,0 +1,56 @@
+name: ESLint Plugin E2E Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    paths-ignore:
+      - compiler/**
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref_name }}-${{ github.event.pull_request.number || github.run_id }}
+  cancel-in-progress: true
+
+env:
+  TZ: /usr/share/zoneinfo/America/Los_Angeles
+
+jobs:
+  # ----- TESTS -----
+  test:
+    name: ESLint v${{ matrix.eslint_major }} E2E Test
```

```suggestion
    name: v${{ matrix.eslint_major }}
```

Looks a bit cleaner :D